html/news/Cinema_Perpetuum_Mobile_2014_files/newsStyle.css

```diff
@@ -0,0 +1,138 @@
+.container-fluid {
+    margin: -8px;
+    padding: 0px;
+}
+
+.b-newsblock {
```

Comment:
b- prefix is used in BEM methology, filmfest is not using it. So may be there is need to remove that prefixes
(it's more about about style and quality of work then work itself - so if you want - you can correct it)